### PR TITLE
Added Landing, Register, Login pages & Redux store

### PR DESCRIPTION
**Pull request by tarasbashuk**

### CODE REVIEW

**Comment by tarasbashuk** [comment]
[Create login page](https://trello.com/c/a113iT8i/88-create-login-page)

**Comment by tarasbashuk** [comment]
[Create registration page](https://trello.com/c/hFwEzaIf/94-create-registration-page)

**Comment by tarasbashuk** [comment]
[Create redux storage](https://trello.com/c/OjTVc3dB/93-create-redux-storage)

**Comment by tarasbashuk** [comment]
[Create Home Page](https://trello.com/c/ozx0ovRH/92-create-home-page)

**Review by skosinskiy — APPROVED** [review]

File: client/.eslintrc [hunk 1]
```diff
@@ -1,9 +0,0 @@
-{
```
Comment (RostyslavStepanchuk) [inline client/.eslintrc:1]: Why do you delete this?

**Review by RostyslavStepanchuk — APPROVED** [review]

File: client/.eslintrc (hunk 1 above)
Comment (tarasbashuk) [reply client/.eslintrc:1]: Sorry, my bad. I'll return it in 5 min.

**Pull request merged**